### PR TITLE
[PE-4351] Added enterprise menu icons

### PR DESCRIPTION
**Pull request by mattnickles**
@jllr Needed for 1.4.1 release

### CODE REVIEW

**Comment by allyourbotsarebelongtous** [comment]
Build FAILED. See http://jenkins.elasticbox.com/job/chi_pull_request/640/ for more details.

**Comment by DiegoSanjuan** [comment]
test this please

**Comment by allyourbotsarebelongtous** [comment]
Build FAILED. See http://jenkins.elasticbox.com/job/chi_pull_request/641/ for more details.

File: cypress/fixtures/chidata.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
...
...e-outline", "file", "filter", "folder-gear", "folder-open", "folder", "globe-network", "globe-outline", "grid", "heart", "home-outline", "home", "key", "life-ring", "link", "list", "lock-outline", "lock", "logo-amazon", "logo-aws", "logo-azure", "logo-centurylink", "logo-cloudstack", "logo-dimensiondata", "logo-facebook", "logo-github", "logo-google", "logo-hp", "logo-linkedin", "logo-linux", "logo-openstack", "logo-rackspace", "logo-slack", "logo-twitter", "logo-vcloud", "logo-vsphere", "logo-youtube", "logout", "mail-outline", "mail", "map-marker-alt", "map-marker", "map-marker-outline", "menu", "microphone-mute", "microphone", "minus-alt", "minus", "more-vert", "more", "nodes", "number", "pause", "phone", "play", "plus-alt", "plus", "radio-checked", "radio", "refresh", "rocket", "search", "settings", "share", "shield", "square-minus-outline", "square-minus", "square-plus-outline", "square-plus", "star-outline", "star", "stop", "sync", "tag-outline", "tag", "text", "triangle-down", "triangle-left", "triangle-right", "triangle-up", "user", "users", "verified", "video-camera-mute", "video-camera", "visible-off", "visible", "warning", "wifi-outline", "wifi", "wrench", "x-alt", "x"],
```
Comment (jllr) [inline cypress/fixtures/chidata.json]: @mattnickles I'm missing inbox icon here. Please add.

File: src/website/views/components/icon.pug [hunk 2]
```diff
@@ -144,7 +144,7 @@ p.-text By default, icons are the same color as text. Use Chi text color utiliti
...
...le-outline', 'file', 'filter', 'folder-gear', 'folder-open', 'folder', 'globe-network', 'globe-outline', 'grid', 'heart', 'home-outline', 'home', 'key', 'life-ring', 'link', 'list', 'lock-outline', 'lock', 'logo-amazon', 'logo-aws', 'logo-azure', 'logo-centurylink', 'logo-cloudstack', 'logo-dimensiondata', 'logo-facebook', 'logo-github', 'logo-google', 'logo-hp', 'logo-linkedin', 'logo-linux', 'logo-openstack', 'logo-rackspace', 'logo-slack', 'logo-twitter', 'logo-vcloud', 'logo-vsphere', 'logo-youtube', 'logout', 'mail-outline', 'mail', 'map-marker-alt', 'map-marker', 'map-marker-outline', 'menu', 'microphone-mute', 'microphone', 'minus-alt', 'minus', 'more-vert', 'more', 'nodes', 'number', 'pause', 'phone', 'play', 'plus-alt', 'plus', 'radio-checked', 'radio', 'refresh', 'rocket', 'search', 'settings', 'share', 'shield', 'square-minus-outline', 'square-minus', 'square-plus-outline', 'square-plus', 'star-outline', 'star', 'stop', 'sync', 'tag-outline', 'tag', 'text', 'triangle-down', 'triangle-left', 'triangle-right', 'triangle-up', 'user', 'users', 'verified', 'video-camera-mute', 'video-camera', 'visible-off', 'visible', 'warning', 'wifi-outline', 'wifi', 'wrench', 'x-alt', 'x']
```
Comment (jllr) [inline src/website/views/components/icon.pug]: @mattnickles I'm missing inbox icon here. Please add.

File: src/website/views/custom-elements/icon.pug [hunk 3]
```diff
@@ -69,7 +69,7 @@ h2 Additional usage details
...
...le-outline', 'file', 'filter', 'folder-gear', 'folder-open', 'folder', 'globe-network', 'globe-outline', 'grid', 'heart', 'home-outline', 'home', 'key', 'life-ring', 'link', 'list', 'lock-outline', 'lock', 'logo-amazon', 'logo-aws', 'logo-azure', 'logo-centurylink', 'logo-cloudstack', 'logo-dimensiondata', 'logo-facebook', 'logo-github', 'logo-google', 'logo-hp', 'logo-linkedin', 'logo-linux', 'logo-openstack', 'logo-rackspace', 'logo-slack', 'logo-twitter', 'logo-vcloud', 'logo-vsphere', 'logo-youtube', 'logout', 'mail-outline', 'mail', 'map-marker-alt', 'map-marker', 'map-marker-outline', 'menu', 'microphone-mute', 'microphone', 'minus-alt', 'minus', 'more-vert', 'more', 'nodes', 'number', 'pause', 'phone', 'play', 'plus-alt', 'plus', 'radio-checked', 'radio', 'refresh', 'rocket', 'search', 'settings', 'share', 'shield', 'square-minus-outline', 'square-minus', 'square-plus-outline', 'square-plus', 'star-outline', 'star', 'stop', 'sync', 'tag-outline', 'tag', 'text', 'triangle-down', 'triangle-left', 'triangle-right', 'triangle-up', 'user', 'users', 'verified', 'video-camera-mute', 'video-camera', 'visible-off', 'visible', 'warning', 'wifi-outline', 'wifi', 'wrench', 'x-alt', 'x']
```
Comment (jllr) [inline src/website/views/custom-elements/icon.pug]: @mattnickles I'm missing inbox icon here. Please add.

File: test/chi/components/icons.pug [hunk 4]
```diff
@@ -2,7 +2,7 @@
...
...e-outline', 'file', 'filter', 'folder-gear', 'folder-open', 'folder', 'globe-network', 'globe-outline', 'grid', 'heart', 'home-outline', 'home', 'key', 'life-ring', 'link', 'list', 'lock-outline', 'lock', 'logo-amazon', 'logo-aws', 'logo-azure', 'logo-centurylink', 'logo-cloudstack', 'logo-dimensiondata', 'logo-facebook', 'logo-github', 'logo-google', 'logo-hp', 'logo-linkedin', 'logo-linux', 'logo-openstack', 'logo-rackspace', 'logo-slack', 'logo-twitter', 'logo-vcloud', 'logo-vsphere', 'logo-youtube', 'logout', 'mail-outline', 'mail', 'map-marker-alt', 'map-marker', 'map-marker-outline', 'menu', 'microphone-mute', 'microphone', 'minus-alt', 'minus', 'more-vert', 'more', 'nodes', 'number', 'pause', 'phone', 'play', 'plus-alt', 'plus', 'radio-checked', 'radio', 'refresh', 'rocket', 'search', 'settings', 'share', 'shield', 'square-minus-outline', 'square-minus', 'square-plus-outline', 'square-plus', 'star-outline', 'star', 'stop', 'sync', 'tag-outline', 'tag', 'text', 'triangle-down', 'triangle-left', 'triangle-right', 'triangle-up', 'user', 'users', 'verified', 'video-camera-mute', 'video-camera', 'visible-off', 'visible', 'warning', 'wifi-outline', 'wifi', 'wrench', 'x-alt', 'x'];
```
Comment (jllr) [inline test/chi/components/icons.pug]: @mattnickles I'm missing inbox icon here. Please add.

File: test/chi/custom-elements/icons.pug [hunk 5]
```diff
@@ -5,7 +5,7 @@ script(type='module' src='../../js/ce/ux-chi-ce/ux-chi-ce.esm.js')
...
...e-outline', 'file', 'filter', 'folder-gear', 'folder-open', 'folder', 'globe-network', 'globe-outline', 'grid', 'heart', 'home-outline', 'home', 'key', 'life-ring', 'link', 'list', 'lock-outline', 'lock', 'logo-amazon', 'logo-aws', 'logo-azure', 'logo-centurylink', 'logo-cloudstack', 'logo-dimensiondata', 'logo-facebook', 'logo-github', 'logo-google', 'logo-hp', 'logo-linkedin', 'logo-linux', 'logo-openstack', 'logo-rackspace', 'logo-slack', 'logo-twitter', 'logo-vcloud', 'logo-vsphere', 'logo-youtube', 'logout', 'mail-outline', 'mail', 'map-marker-alt', 'map-marker', 'map-marker-outline', 'menu', 'microphone-mute', 'microphone', 'minus-alt', 'minus', 'more-vert', 'more', 'nodes', 'number', 'pause', 'phone', 'play', 'plus-alt', 'plus', 'radio-checked', 'radio', 'refresh', 'rocket', 'search', 'settings', 'share', 'shield', 'square-minus-outline', 'square-minus', 'square-plus-outline', 'square-plus', 'star-outline', 'star', 'stop', 'sync', 'tag-outline', 'tag', 'text', 'triangle-down', 'triangle-left', 'triangle-right', 'triangle-up', 'user', 'users', 'verified', 'video-camera-mute', 'video-camera', 'visible-off', 'visible', 'warning', 'wifi-outline', 'wifi', 'wrench', 'x-alt', 'x'];
```
Comment (jllr) [inline test/chi/custom-elements/icons.pug]: @mattnickles I'm missing inbox icon here. Please add.

**Comment by DiegoSanjuan** [comment]
test this please

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://3397c500c2f86f841699f60e4dec444460a2c216-chi-pr.enterprise.ctl.io" target="_blank"> https://3397c500c2f86f841699f60e4dec444460a2c216-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/649/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://b286694ce5957adcc629ffab2ba58b41fbbc5f18-chi-pr.enterprise.ctl.io" target="_blank"> https://b286694ce5957adcc629ffab2ba58b41fbbc5f18-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/650/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
The following instances are being terminated: https://ops.elasticbox.com/services/instances/i-8anbys, https://ops.elasticbox.com/services/instances/i-hboouu

**Pull request merged**